README.md

```diff
@@ -71,6 +71,35 @@ For example:
 - Amsterdam Schema is used to validate the table row JSON Schema,
   with a _meta-schema_ (a JSON Schema to verify a JSON Schema).
 
+## Dataset versioning
+If different versions of a dataset exist, we create a versions.json file in the parent directory
+  of the dataset. This versions file will contain the id of the dataset, a `versions` object, with the
+  name and location of the dataset.json files, and a `default` property
+  with the location of the default version.
+  So for our `bag` dataset example the folder structure might look like this:
+  - bag /
+    - versions.json
+    - v2.0.0 /
+      - dataset.json
+      - buildings.json
+      - addresses.json
```

Comment:
deze structuur is vast elders al vastgesteld na veel overleg, maar toch heb ik hier nog een bezwaar:

de "dataset.json" heeft geen andere extensie dan de rest. Nu zal de tabel wel niet snel "dataset" heten, maar toch is dat jammer. Of, heeft staat de bestandsnaam helemaal los van de filenaam, en zou ik het iedere willekeurige naam kunnen geven?